### PR TITLE
fix: fix translations + update readme

### LINKED ISSUE #26: Validation messages not loaded from translations
Hi! 👋 

First, thank you for this package :) It works great, but it seems that validation messages aren't loaded from translations: 

Screenshot 2019-06-02 at 20 57 17

I fixed the issue by changing the `Enum` rule code as follow:

Screenshot 2019-06-02 at 20 58 46

Screenshot 2019-06-02 at 20 59 41

Am I doing something wrong?

Thanks for your help!

### PR DESCRIPTION
**Pull request by atymic**
It seems that the validation rules were not pulling in translations correctly.
At least in 5.7 & 5.8, the returned message would just be the translation key.

This fixes that by correcting the keys + fixing the tests.
I also added instructions to the readme on how to publish the translations.

I did consider the BC break by renaming the file that holds the messages, but since it wasn't working anyway i'm not sure it's a problem. (though happy to keep it as before if required).

Fixes #26

### CODE REVIEW

File: src/Rules/CountryCode.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function passes($attribute, $value): bool
 
     public function message(): string
     {
-        return __('validation.country_code', [
+        return __('validationRules::messages.country_code', [
```
Comment (freekmurze) [inline src/Rules/CountryCode.php]: Let's keeping use the `validation` file here and for the other rules.

Comment (atymic) [reply src/Rules/CountryCode.php]: Currently the file is called `validationRules` so should it be
```
'validationRules::validationRules.country_code'
```
or 
```
'validationRules::validation.country_code'
```

Thanks

**Comment by freekmurze** [comment]
Tests seem to be failing for this one, could you fix that?

**Comment by thibaultlavoisey** [comment]
@atymic Let me know if I can help you in any way :)

**Comment by atymic** [comment]
I'll have a look today and fix the tests :)

File: src/Rules/CountryCode.php (hunk 1 above)
Comment (atymic) [reply src/Rules/CountryCode.php]: @freekmurze sorry, could you let me know which one you prefer?
Thanks

**Comment by atymic** [comment]
Tests have been fixed, the translation rules were being loaded in the global namespace rather than the package namespace.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**